Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,19 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  return tasks
+    .filter(task => task.duration >= 120) // when i do the filter after the map i get an empty array
```

Comment:
You could do the filter after the map. But you have to compare with a different number. Can you think of what number that should be?

Reply:
yes it will be 2. because of the hourly expression. I try it but i get an empty array